manifests/cloud-config/paas-cf-cloud-config.yml

```diff
@@ -212,6 +212,17 @@ vm_types:
     - ((terraform_outputs_elasticache_broker_clients_security_group))
     - ((terraform_outputs_default_security_group))
 
+- name: notify_cell
```

Comment:
Can we not call them notify cells? it's a type and more generic one could be useful elsewhere?

Reply:
+1

Naming it for what it is (a cell type with more CPU power) would be better

Reply:
I was going for the Ronseal approach whilst testing was happening